README.md

```diff
@@ -7,11 +7,31 @@ fuse-wsfs combines libwebsockets and libfuse. It allows ot attach a remote files
 
 ## Contents
 
--   [Workflow and API](#Workflow-and-API)
+-   [Motication](#Motivation)
+-   [Concept](#Concept)
+-   [Similar Projects](#Similar-Projects)
+-   [API](#API)
 -   [Build and run](#Build-and-run)
 -   [Dependencies](#Dependencies)
 
-## Workflow and API
+## Motivation
+
+Many embedded devices, such as smart home or [IoT](https://en.wikipedia.org/wiki/Internet_of_things) devices are very limited regarding to their (non-volatile) memory resources. Such devices are typically comprised of an embedded linux and a small web server, providing an interface for maintenance purposes.
+
+Some use cases, such as firmware update, require to transfer (larger) files to the device. The firmware file is often stored multiple times on the device:
+
+1.  cached by the web server, e.g. [lighttpd](https://redmine.lighttpd.net/boards/2/topics/3451)
+2.  copied to locally, e.g. /tmp
+3.  uncompressed, also to /tmp
+
+Techniques like [SquashFS](https://en.wikipedia.org/wiki/SquashFS) helps to avoid the third step, since the upgrade file can be mounted directly. [RAUC](https://rauc.io/) shows the use of SquashFS within an update facility.  
```

Comment:
Multiple techniques mentioned -> use of plural
```suggestion
Techniques like [SquashFS](https://en.wikipedia.org/wiki/SquashFS) help to avoid the third step, since the upgrade file can be mounted directly. [RAUC](https://rauc.io/) shows the use of SquashFS within an update facility.  
```

Reply:
Thanks, will be fixed